.claude/rules/dependency-version-compatibility.md

```
@@ -0,0 +1,132 @@
+## 既知のバージョン互換性情報
+
+### tower-lsp と lsp-types
+
+| tower-lsp | lsp-types | 備考 |
+|-----------|-----------|------|
+| 0.20 | 0.94 | 安定版。`SemanticTokensResult::Tokens` バリアント |
+| 0.20.x | 0.94.x | マイナーアップデートは互換 |
+
+### このプロジェクトのクレート依存関係
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)